Kvantum/themes/kvthemes/KvArc/KvArc.kvconfig

```diff
@@ -230,10 +230,10 @@ indicator.element=tree
 
 [HeaderSection]
 inherits=PanelButtonCommand
-frame.top=2
-frame.bottom=2
-frame.left=2
-frame.right=2
+frame.top=0
+frame.bottom=1
+frame.left=0
+frame.right=1
 
```

Comment:
This should be
````
frame.left=1
frame.right=0
````
for header separators to be drawn because their width is that of left frame.